index.html

```diff
@@ -249,14 +249,14 @@ <h1>The Registration Process</h1>
 Any method name SHOULD avoid generic terms such as "mymethod" or "registry".
       </li>
       <li>
-If there are copyright, trademark, or any intellectual property rights
-concerns, the addition and use MUST be authorized in writing by the intellectual
-property rights holder under a
-<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a>
-license. Examples include DID Methods that use trademarked brand names,
-property names that utilize the titles of copyrighted works, and patented
-technology that would cause the use of the extension to require licensing a
-patent.
+        Any DID Method submission MUST not impose any copyright, trademark, or intellectual property rights concerns on the W3C, DID Method implementers, or users.  To that end, the following apply:<BR>
+
+          <OL>
```

Comment:
- Convention is to use lower case tags.
- Style in this doc is generally tags on their own line, content flush left on it's own line and wrapped at 80 chars.
- Move `<ol>` indents back two spaces to align with the formatting style.

Reply:
Thank you for letting me know.  I uploaded some updates.  Does it look better now?